crypto/x509/x509_vfy.c

```diff
@@ -186,6 +191,21 @@ static int verify_cb_crl(X509_STORE_CTX *ctx, int err)
     return ctx->verify_cb(0, ctx);
 }
 
+#ifndef OPENSSL_NO_OCSP
+/*
+ * Inform the verify callback of an error, OCSP-specific variant.  Here, the
```

Comment:
It should be documented somewhere that the verification callback function may also be called on OSCP response errors if the `X509_V_FLAG_OCSP_RESP_CHECK` or `X509_V_FLAG_OCSP_RESP_CHECK_ALL` flag is set.

Reply:
Added comment.

Reply:
Where are you going to add it to, with which text?

Reply:
Is committed and pushed